shell/platform/windows/flutter_window_winuwp.cc

```diff
@@ -9,6 +9,10 @@ namespace flutter {
 // Multipler used to map controller velocity to an appropriate scroll input.
 static constexpr double kControllerScrollMultiplier = 3;
 
+// TODO: Determine pointer ID in
+// OnPointerPressed/OnPointerReleased/OnPointerMoved to support multi touch
```

Comment:
Please file an issue for this and add the link here as per this part of the style guide:

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-checking-in-comments-that-ask-questions